Remotion/Web/Core/Res/Themes/NovaGray/HTML/TreeView.css

```diff
@@ -189,6 +189,15 @@ span.treeViewNode ul
   right: auto;
 }
 
+span.treeViewNodeBadge
```

Comment:
could you please a screenshot before and after the badge changes and with/without the badge. Want to quickly make sure there's no layout changes in novagray.

Reply:
![image](https://user-images.githubusercontent.com/5082673/134879877-86a5f08c-3992-4638-9000-1e7954705088.png)

Reply:
i guess there is still a problem with the spacing

Reply:
fixed version:
![image](https://user-images.githubusercontent.com/5082673/134883309-59af10e3-3549-49ce-836c-39815f82d5a2.png)